vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ImageService.java

```diff
@@ -62,4 +78,23 @@ public ImageData getImageDimensions(IVSImage image, int width, int height) {
         Float newHeight = (new Float(width) / new Float(imageWidth)) * new Float(imageHeight);
         return new ImageData(newHeight.intValue(), width);
     }
+    
+    
+    /**
+     * Method to return the required attributes of a requested page  
+     * 
+     * @param currPage current page requested from ui
+     * @return map of required attributes
+     */ 
+    @Override
+    public Map<String, Object> getImageListingAttr(int currPage) {
```

Comment:
spell out the the variable name. there is really no real reason to abbreviate current

Reply:
also, bad method name. Why image listing attr? which attributes?  we want a list of images according to the parameters we hand to the method.